.gitattributes

```diff
@@ -22,17 +22,17 @@
 *.ttf  binary
 
 # Ignore some meta files when creating an archive of this repository
+# We do not ignore any content, because this repo represents the 
+# `yiisoft/yii2-dev` package, which is expected to ship all tests and docs.
```

Comment:
We don't need this part, it's about Yii 2.

Reply:
done.